# grasp point
float64[] g
---